docs/disks.md

```diff
@@ -0,0 +1,41 @@
+# RAID setup
```

Comment:
@jakogut should we rename the file to `raid.md`? The current name seems to generic and the content is very RAID specific.

Reply:
@alexgg Yes, agreed, I'll rename this. Should we also consider publishing this information elsewhere, or do you think a Google search will turn this up eventually?

Reply:
@jakogut definitely a blog post is a good idea to give visibility to the feature. And probably it's something that could also be linked into the balenaOS product documentation but I don't think there is currently a good place for it. We can discuss it with @vipulgupta2048